llm/main.py

```diff
@@ -204,21 +206,19 @@ def main(cfg):
         schedulers=scheduler,
         max_duration=cfg.max_duration,
         eval_interval=cfg.eval_interval,
-        eval_subset_num_batches=cfg.eval_loader.get('eval_subset_num_batches',
-                                                    -1),
+        eval_subset_num_batches=cfg.eval_subset_num_batches,
         progress_bar=cfg.progress_bar,
         log_to_console=cfg.log_to_console,
-        console_log_interval='1ba',
+        console_log_interval=cfg.console_log_interval,
         loggers=loggers,
         callbacks=callbacks,
         precision=cfg.precision,
-        algorithms=algos,
-        device_train_microbatch_size=cfg.get('device_train_microbatch_size', 'auto'),
+        algorithms=algorithms,
+        device_train_microbatch_size=cfg.device_train_microbatch_size,
         fsdp_config=fsdp_config,  # type: ignore
         save_folder=cfg.get('save_folder', None),
         save_interval=cfg.get('save_interval', '1000ba'),
-        save_num_checkpoints_to_keep=cfg.get('save_num_checkpoints_to_keep',
-                                             -1),
+        save_num_checkpoints_to_keep=cfg.get('save_num_checkpoints_to_keep', -1),
```

Comment:
I feel like this is one of those edits a linter made ie 
`        save_num_checkpoints_to_keep=cfg.get('save_num_checkpoints_to_keep', -1),`
is 81 chars long so a linter split the line. You're undoing it. When whoever (ie Davis) runs a linter in the future, they'll redo it.

feel free to ignore this cmt.

Reply:
I think i need to install the linter locally to fix this, will do that